src/Hng.Application/Features/Organisations/Handlers/GetUsersOrganizationsHandler.cs

```diff
@@ -27,8 +27,7 @@ public GetUsersOrganizationsHandler(IRepository<Organization> organizationReposi
         public async Task<SuccessResponseDto<List<OrganizationDto>>> Handle(GetAllUsersOrganizationsQuery request, CancellationToken cancellationToken)
         {
             var userId = await _authenticationService.GetCurrentUserAsync();
-            var user = await _userRepository.GetBySpec(x => x.Id == userId, u => u.Organizations);
-            var orgs = user.Organizations.ToList();
+            var orgs = await _organizationRepository.GetAllBySpec(x => x.OwnerId == userId);
```

Comment:
This was to get all organizations the user belongs to, the new filter you added would get only the ones the user created, revert this